datalake/backend/processing/check_metadata_content/task.py

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env python3
+import sys
```

Comment:
Is this script following and checking all child metadata files ? See: https://confluence.linz.govt.nz/pages/viewpage.action?pageId=93566252

Reply:
Not yet, no.